components/date-picker/style/MonthPanel.less

```diff
@@ -18,10 +22,14 @@
   .calendarPanelHeader(~"@{calendar-prefix-cls}-month-panel");
 }
 
+.@{calendar-prefix-cls}-month-panel-body {
+  height: ~"calc(100% - 34px)";
```

Comment:
It works fine in IE9

![image](https://cloud.githubusercontent.com/assets/3580607/24736135/e223dacc-1ab8-11e7-88d8-3952cc620926.png)